resources/icons/16px/pt-icon-snippet.svg

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this `snippet` icon? I don't see it on blueprintjs.com right now.

Reply:
i think that's why @pkwi is removing it: it is actually unused.

Reply:
also it only appeared in 16px, not 20px